targets/lenovo-x1/everything.nix

```diff
@@ -124,140 +101,18 @@
               host.networking.enable = true;
               virtualization.microvm.netvm = {
                 enable = true;
-                extraModules = let
-                  configH = config;
-                  netvmPCIPassthroughModule = {
-                    microvm.devices = lib.mkForce (
-                      builtins.map (d: {
-                        bus = "pci";
-                        inherit (d) path;
-                      })
-                      configH.ghaf.hardware.definition.network.pciDevices
-                    );
-                  };
-                in
-                  [netvmPCIPassthroughModule]
-                  ++ netvmExtraModules;
+                extraModules =
+                  import ./netvmExtraModules.nix {inherit lib config pkgs microvm hostConfiguration;};
```

Comment:
Actually I'm not sure, it looks like you pass in the host configuration twice, first as a config, then as a hostConfiguration.config

Reply:
Maybe we could have a single parameter `configH` or `configHost` instead of `config` and `hostConfiguration`.

And then here we say inside the `import ./netvmExtraModules.nix { inherit lib pkgs microvm; configHost = config };`, to make it very explicit